addons/mail/static/src/messaging/component/message/message.scss

```diff
@@ -332,6 +332,15 @@
     }
 }
 
+.o_Message_notificationIconContainer {
+    color: $gray-600;
```

Comment:
didn't we say we prefer `gray('<value>')` over `$gray-<value>`? This variable is just a useless alias, and even its naming is too similar with original one, so that's a useless use of variable IMO

Reply:
I don't remember but ok.

The variable is not an alias though it is the original value.
The function is the alias, but I guess it could be useful in some kind of customization scheme.

Reply:
You're right... Now I don't know which one is better... Maybe keep `gray()` for consistency, as you said it could be useful for some customization maybe?